docs/setup/developer.md

```diff
@@ -2,6 +2,18 @@
 
 ## Setting up `Python` Virtual Environment (VE)
 
+### Using `uv` (stringly recommended)
+Once you have installed `uv`, in the root of your source tree you can:
+
+Create the venv (one-off task)
+```
+uv venv
+```
+Enter the venv (for the lifetime of the shell)
+```
+source .venv/bin/activate
+```
+
 ### Using conda
```

Comment:
I'd take out the conda and venv options, and only have `uv` as the only way that we document

Reply:
The GAE instructions also had old references to conda. Mostly replaced with link to Slab doc.